.github/workflows/smoke-tests.yaml

```diff
@@ -45,6 +45,7 @@ jobs:
           path: 'target/surefire-reports'
       - name: Bee Destroy
         uses: broadinstitute/workflow-dispatch@v4
+        if: always()
```

Comment:
I'm curious -- if Bee fails to create, what is there to destroy?

Reply:
> I'm curious -- if Bee fails to create, what is there to destroy?

I don't understand the internals completely, but it's a Sherlock thing. A failed bee creation can still end up with a Sherlock entry. The delete job ensures that Sherlock is purged of the failed bee.

Reply:
Thanks for the explanation!